data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -243,13 +243,32 @@
 }
 
 SPANISH_CATEGORY_MAPPINGS = {
-    "(?i).*DETERGENTE.*": "Home Care",
-    "(?i).*COSMETICO.*": "Personal Care",
+    "(?i).*AUTOMOTOR.*": "Other",
```

Comment:
I also saw some Spanish category mappings in latam_mexico_transform_functions.py. Could you please make sure to bring some of the common ones (usable for other LATAM countries) into here? Thank you.